packages/tasks/src/model-libraries.ts

```diff
@@ -506,6 +506,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"cfg.json"`,
 	},
+	sapiens: {
```

Comment:
I have already addressed this. I'll open PRs today, this will not take effect immediately anyway so it can be merged

Reply:
Since the PR is for the `Sapiens` library, it'd probably make sense to wait till the merge your Hub PRs and then merge this PR. Otherwise, this PR/ downloads won't make much sense.

(you know this collab/ authors better, I'm commenting purely from hub perspective)

Reply:
I just send them a script to merge all let's see